src/app/session/[type]/Steps/helper.ts

```diff
@@ -283,6 +278,9 @@ export const handleBatchFields = (
     fieldsSchema.parentBatch.hide = false;
   }
 
+  if (value === Platform.NoPlatform) {
+    form.setValue('isRedirection', false, { shouldDirty: true });
```

Comment:
if value changes, set redirection to true

Reply:
Added